tensorbay/label/tests/test_label.py

```diff
@@ -0,0 +1,451 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+import pytest
+
+from ...geometry import (
+    Box2D,
+    Box3D,
+    Keypoint2D,
+    Keypoints2D,
+    Polygon2D,
+    Polyline2D,
+    Quaternion,
+    Transform3D,
+    Vector2D,
+    Vector3D,
+)
+from .. import (
+    Classification,
+    LabeledBox2D,
+    LabeledBox3D,
+    LabeledKeypoints2D,
+    LabeledPolygon2D,
+    LabeledPolyline2D,
+    LabeledSentence,
+    LabelType,
+    Word,
+)
+
+_CATEGORY = "None"
+_ATTRIBUTES = {"key": "value"}
```

Comment:
Set _CATEGORY and _ATTRIBUTES the same as _CLASSIFICATION_DATA, so you can reuse it in test_dumps() after